lib/view_component/render_monkey_patch.rb

```diff
@@ -3,7 +3,7 @@
 module ViewComponent
   module RenderMonkeyPatch # :nodoc:
     def render(options = {}, args = {}, &block)
-      if options.respond_to?(:render_in)
+      if options.respond_to?(:render_in) && Rails.version.to_f < 6.1
```

Comment:
Should we do just a tiny bit of cleanup here? Perhaps a static method like `ViewComponent::apply_render_monkey_patch(options)` that we use is these two locations?

Reply:
I agree that we should just wrap the monkey patch entirely once we ship `v2`. As I plan to do that in the next week or so, I'm not going to sweat this temporary duplication 😄